### ISSUE TITLE
Unity serialization and readonly attribute

### ISSUE THREAD

**Issue by DanielAdolfsson** [jacobdufault/fullinspector#67]
Unity doesn't seem to support serializing readonly fields in a meaningful way, and thus `InspectedType.IsSerializedByUnity` returns false positives when encountering them.

This may or may not be considered a bug, and whether readonly fields should be serialized or not is a whole other discussion. But since the comment clearly states "However, it will _not_ return true if the type cannot be serialized by unity.", I would consider it a bug.

To reproduce:

``` C#
public readonly float Distance = 1.5f;
```

Suggested fix in `InspectedType.IsSerializedByUnity`:

``` C#
if (property.MemberInfo is FieldInfo && (property.MemberInfo as FieldInfo).IsInitOnly) {
    return false;
}
```
